schema/column.go

```diff
@@ -46,7 +25,7 @@ type Column struct {
 	// Value Type of column i.e String, UUID etc'
 	Type ValueType `json:"type,omitempty"`
 	// Description about column, this description is added as a comment in the database
-	Description string `json:"description,omitempty"`
+	Description string `json:"-"`
```

Comment:
Why is description being removed from the JSON?

Reply:
Just saving some bandwidth, I dont think we use it anywhere. but I can get it back.

Reply:
Yeah not a big deal, like you say, I don't think we are using this anywhere right now.